conjure-codegen/src/example_types/another/test_service.rs

```diff
@@ -13,14 +13,17 @@ impl<T> TestServiceAsyncClient<T>
 where
     T: conjure_http::client::AsyncClient,
 {
-    #[doc = "Returns a mapping from file system id to backing file system configuration."]
+    ///Returns a mapping from file system id to backing file system configuration.
     pub async fn get_file_systems(
         &self,
         auth_: &conjure_object::BearerToken,
     ) -> Result<
-        std::collections::BTreeMap<String, super::super::product::datasets::BackingFileSystem>,
-        conjure_http::private::Error,
-    > {
+            std::collections::BTreeMap<
+                String,
+                super::super::product::datasets::BackingFileSystem,
+            >,
+            conjure_http::private::Error,
+        > {
```

Comment:
Fixed in prettyplease 0.1.11.

```suggestion
        std::collections::BTreeMap<
            String,
            super::super::product::datasets::BackingFileSystem,
        >,
        conjure_http::private::Error,
    > {
```

Reply:
Bumped, thanks!